### DISCUSSION TITLE
Error after running playbook

### DISCUSSION THREAD

**Discussion by Shinz0n** [aristanetworks/avd#1168] [Q&A] [answered]
Hi,

I recently set up an AVD environment (https://avd.sh/en/latest/docs/getting-started.html) and made some slight changes at the demo "ansible-avd-cloudvision-demo" to fit my needs (GNS3 running vEOS 4.26.2F with 2 spines and 3 leafs). However after a playbook run, I do get the following error that I'm unable to solve:

`ansible-playbook playbooks/dc1-fabric-deploy-cvp.yml`

`
TASK [arista.avd.eos_config_deploy_cvp : Configure devices on cv_server] **************************************************************
fatal: [cv_server]: FAILED! => changed=false
  msg: Device input data are not compliant with module.
`

See full task output [task_output.txt](https://github.com/aristanetworks/ansible-avd/files/7097333/task_output.txt)

I'm running:
CVP 2021.1.1
arista.avd 2.2.1
arista.cvp 3.1.2
ansible 2.10.12

Appreciated for any help.

**Comment by titom73** [comment]
Hello @Shinz0n 

Thanks for opening issue and your report.

The repository you are pointed is compatible with AVD and CVP collections in version 2.x and is not yet ready to support v3.x collections. This change will be done once AVD will be released in v3

To use v2 collections, please use [following commands](https://github.com/arista-netdevops-community/ansible-avd-cloudvision-demo#getting-started):

```bash
$ ansible-galaxy collection install arista.avd:==2.0.0
$ ansible-galaxy collection install arista.cvp:==2.1.2
```

As it is a discussion related to a side project, I will move this issue to Discussion section.

Kind Regards,

**Comment by Shinz0n** [comment]
Hi @titom73 

I have downgraded the corresponding collections. Unfortunately I do get the same error:
```

TASK [arista.avd.eos_config_deploy_cvp : Configure devices on cv_atd1] ****************************************************************
fatal: [cv_atd1]: FAILED! => changed=false
  msg: Device input data are not compliant with module.
```

```
/home/avd/.ansible/collections/ansible_collections
Collection        Version
----------------- -------
ansible.netcommon 2.4.0
ansible.utils     2.4.0
arista.avd        2.0.0
arista.cvp        2.1.2
arista.eos        3.0.0
```

Is there a way to narrow down which part of the data is not compliant?

**Reply by titom73** [reply]
It seems output is from another [demo repository](https://github.com/arista-netdevops-community/atd-avd) here as it is now `cv_atd1` and not `cv_server`.

I was able to run playbook from ansible-avd-cloudvision-demo with the following change:

- remove all arista.avd and arista.cvp collections
- Re-install arista.avd. only in version 2.2.1 (CV collection will come automatically)

If you still have the same error, please post content of your `intended/structured_configs/cv/cv_server.yml` for analysis

**Comment by Shinz0n** [comment, marked as answer]
Sorry @titom73 for the ambiguous output. Acutally I played around with some other inventories at once. Long story short: I did a full wipe as suggested and now use the "atd-avd" demo instead of the shipped "ansible-avd-cloudvision-demo" without any problems. Thank you.
